lib/sync/worker.js

```diff
@@ -1,6 +1,27 @@
 var syncUtil = require('./util');
 var debug = syncUtil.debug;
 var metrics = require('./sync-metrics');
+var backoff = require('backoff');
+
+function initBackoff(backOffOpts) {
+  var bo;
+  if (backOffOpts.strategy === 'exp') {
+    bo = new backoff.ExponentialStrategy({initialDelay: backOffOpts.min, maxDelay: backOffOpts.max});
+  } else if (backOffOpts.strategy === 'fib') {
+    bo = new backoff.FibonacciStrategy({initialDelay: backOffOpts.min, maxDelay: backOffOpts.max});
+  } else {
+    //if no valid value is given, just return the default minimum delay. Can be used to turn off backoff.
```

Comment:
Should this throw an exception if an invalid strategy is configured?
It might be safer that a misspelled config. For example if someone sets the strategy to "FIB", it won't use fibonnaci. Instead it will use the 'minimum' strategy. This might be a cause of confusion